code/modules/maps/weather/classm.dm

```diff
@@ -11,7 +11,7 @@
 
 
 /datum/time/classm
-	seconds_in_day = 3 HOURS
+	seconds_in_day = 10 HOURS
```

Comment:
understandable for the ball, but should this be a permanent change?

Reply:
Yes, day cycles were stuck to permanently nighttime at 3-5 hours